setup.cfg

```diff
@@ -0,0 +1,57 @@
+[metadata]
+name = kbmod
+authors = [
+    { name = "KBMoD Developers", email="tmp@email.email"},
+    ]
```

Comment:
We need to fill in some authors, but I wasn't sure what the base way of doing that was. We can list everyone individually, but that list might grow substantial (a little bit already is) or we can just write "KBMoD Developers" without an email. This is how all the other projects (astropy numpy etc...) did it.

Reply:
Thanks for the votes! "KBMoD Developers" wins.